### ISSUE TITLE
Create mock data in raw JavaScript

### ISSUE THREAD

**Issue by ericbutera** [ericbutera/800linkdemo#8]
Before starting on the C# side of things with webapi requests, create a few mocked objects in the page that contain the list of calls & maybe some sample filters.  After this is completed begin work on WebApi integration.

**Comment by ericbutera** [comment]
This is already done. See #2 for initial creation and #4 for the addition of extensions and rename phone to caller per requirements of demo.
